src/humanize/filesize.py

```diff
@@ -5,7 +5,7 @@
 suffixes = {
     "decimal": ("kB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"),
     "binary": ("KiB", "MiB", "GiB", "TiB", "PiB", "EiB", "ZiB", "YiB"),
-    "gnu": "KMGTPEZY",
+    "gnu": ("K", "M", "G", "T", "P", "E", "Z", "Y"),
```

Comment:
This is the only change to the code that is made by this PR, to simplify the typing of the `suffixes` dictionary slightly. But there is no change to behavior.

Reply:
👍 And this makes the code look a bit more consistent.